addons/api/addon/workers/utils/schema.js

```diff
@@ -60,8 +51,60 @@ END;
 CREATE TRIGGER IF NOT EXISTS target_ad AFTER DELETE ON target BEGIN
     INSERT INTO target_fts(target_fts, rowid, id, type, name, description, address, scope_id, created_time)
     VALUES('delete', old.rowid, old.id, old.type, old.name, old.description, old.address, old.scope_id, old.created_time);
+END;`;
+
+const createAliasTables = `
+CREATE TABLE IF NOT EXISTS alias (
+    id TEXT NOT NULL PRIMARY KEY,
+    type TEXT NOT NULL,
+    name TEXT,
+    description TEXT,
+    destination_id TEXT,
+    value TEXT,
+    scope_id TEXT NOT NULL,
+    created_time TEXT NOT NULL,
+    data TEXT NOT NULL
+);
+CREATE INDEX IF NOT EXISTS idx_alias_created_time ON alias(created_time DESC);
```

Comment:
Note this index is only on created time as we don't include filter by scope normally.